addons/mail/static/src/models/attachment_image/attachment_image.js

```diff
@@ -61,7 +61,7 @@ registerModel({
             if (this.attachmentList.composerView) {
                 return 50;
             }
-            if (this.attachmentList.chatter) {
+            if (this.attachmentList.attachmentBoxView && this.attachmentList.attachmentBoxView.chatter) {
```

Comment:
Do we need to guard this? `AttachmentImage/attachmentList` is required and `AttachmentList/attachmentBoxView` is readonly but kinda required as it's an identifying field.

I would expect no guard, but maybe technically it's possible to have `undefined`?

Reply:
Compute order is not guaranteed so if the concerned fields are computed themselves it's necessary.

I'm usually very careful with not adding guard where it makes no sense (required field in code in general), but in computes there's no way around it.

Reply:
Here it's strange because `attachmentList` is the owner and `attachmentBoxView` is its owner, so it should indeed never happen that those are undefined here.

Reply:
Ok that's the solution actually, `attachmentBoxView` is owner, but in OR.

Reply:
So guard on `attachmentBoxView` is needed, but guard on `chatter` is probably overkill as it is sole identifying field.

Reply:
So remove guard on chatter if possible?

Reply:
I don't have issue and the runbot greened the PR so I guess we don't need it.